platform/source/lib/ax/platform/spot_instance_option_manager.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
I'd prefer have a package for asg / minion management related objects rather than squeezing everything under ax.platform

Reply:
Good idea. Created package minion_management and moved SpotInstanceOptionManager under that.